src/sick_tim_common.cpp

```diff
@@ -51,10 +51,9 @@ std::string SickTimCommon::getNamespaceStr()
     return ns;
 }
 
-SickTimCommon::SickTimCommon(AbstractParser* parser, double expected_fps,
-  double fps_tolerance):
+SickTimCommon::SickTimCommon(AbstractParser* parser, marble::OutputDiagnosticParams output_scan_params):
     output_scan_diagnostic_(nullptr), generic_sopas_diagnostic_(nullptr), updater_(nullptr),
-    parser_(parser), expected_fps_(expected_fps), fps_tolerance_(fps_tolerance)
+    parser_(parser), output_scan_params_(output_scan_params)
```

Comment:
nit: Looks like `output_scan_params_` doesn't need to be a member variable, as it is only every used in here in the constructor.

Reply:
Changed